design/search.go

```diff
@@ -71,4 +123,26 @@ var _ = a.Resource("search", func() {
 		a.Response(d.OK)
 		a.Response(d.InternalServerError, JSONAPIErrors)
 	})
+
+	a.Action("codebases", func() {
+		a.Routing(
+			a.GET("codebases"),
+		)
+		a.Description("Search by URL")
+		a.Params(func() {
+			a.Param("url", d.String)
+			a.Param("page[offset]", d.String, "Paging start position") // #428
+			a.Param("page[limit]", d.Integer, "Paging size")
+			a.Required("url")
+		})
+		a.Response(d.OK, func() {
+			a.Media(searchCodebaseList)
```

Comment:
Any reason to not just reuse the code list from code?

Reply:
hum, right, let me check that indeed